protocol-configuration/mainnetAddresses.ts

```diff
@@ -1576,6 +1586,16 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0xD3D13a578a53685B4ac36A1Bab31912D2B2A2F36',
     category: AddressCategory.External
   },
+  voltFeiSwapContract: {
+    artifactName: 'OtcEscrow',
+    address: '0xeF152E462B59940616E667E801762dA9F2AF97b9',
+    category: AddressCategory.External
+  },
+  voltFusePCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0xFeBDf448C8484834bb399d930d7E1bdC773E23bA',
+    category: AddressCategory.PCV
+  },
```

Comment:
Tbh I don't love the intermingling here. Lets think about having a parallel mainnetAddresses configuration in volt or even rewriting the proposal flow in forge

Reply:
I already did a simulation in the volt repo with forge, but it's not the same thing as writing out the calldata steps here. https://github.com/volt-protocol/volt-protocol-core/pull/70/files